rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/BundleResource.java

```diff
@@ -153,6 +154,10 @@ public TypeDetail getTypeExplicitVersion(String symbolicName, String version, St
     @Override
     public BundleInstallationRestResult remove(String symbolicName, String version, Boolean force) {
         ManagedBundle b = lookup(symbolicName, version);
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.MODIFY_CATALOG_ITEM,  Entitlements.StringAndArgument.of(symbolicName+(Strings.isBlank(version) ? "" : ":"+version), "delete"))) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to remove catalog items",
```

Comment:
Message should say `User '%s' is not authorized to remove catalog item '%s:%s'`
Please also check the parameters for the templated message